### DISCUSSION TITLE
Allow custom display names for auth providers

### DISCUSSION THREAD

**Discussion by MoltenCoffee** [directus/directus#10673] [Feature Requests]
At the moment, when configuring auth/SSO providers, the name, as displayed on the frontpage, is based on the items in "AUTH_PROVIDERS", and seems to assume capitalization should be done on every first letter.
Whilst correct for most cases, this could for some (custom) auth providers generate incorrect display names.

I would like to propose a "display name" feature to add to auth providers, which enables the display name to be different from the canonical name of the provider, allowing for interpunction or alternative casing:

```sh
AUTH_PROVIDERS="someProvider"

AUTH_SOMEPROVIDER_DISPLAY_NAME="some Provider"
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
